packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -82,11 +82,11 @@ class ElevatedButton extends ButtonStyleButton {
   ///
   /// The [icon] and [label] arguments must not be null.
   factory ElevatedButton.icon({
-    Key key,
-    required VoidCallback onPressed,
-    VoidCallback onLongPress,
-    ButtonStyle style,
-    FocusNode focusNode,
+    Key? key,
+    required VoidCallback? onPressed,
+    VoidCallback? onLongPress,
+    ButtonStyle? style,
+    FocusNode? focusNode,
     bool autofocus,
     Clip clipBehavior,
```

Comment:
I think `autofocus` and `clipBehaviour` should be nullable as well
https://github.com/flutter/flutter/blob/dde20448f7bce0b98da63a128488100b0bd495e5/packages/flutter/lib/src/material/elevated_button.dart#L377

Reply:
Yes, you are right. we need null safety here as well
I have done this.
Thanks

Reply:
```Hmm, looks like autofocus and clipBehaviour can be nullable only in theory...```
I have removed null safety from clipBehaviour and autofocus.
Thanks